styles.css

```diff
@@ -0,0 +1,70 @@
+
+html {
+    background-image: url('sunray.jpeg');
+    background-size: cover;
+}
+
+.boxone{
+    display:flex; 
+    justify-content: space-evenly;
+    font-weight: bold;
+    font-size: 50px;
+    color: rgb(255, 36, 145);
+}
+.pics{
+    display:flex;
+    justify-content: space-evenly;
+}
+
+.border {
+    border: 6px solid rgb(238, 36, 238);
+    border-radius: 30px;
+  }
+
+h1{
+    color: rgb(228, 50, 228);
+    font-size: 10em;
+    display:flex;
+    justify-content: space-evenly;
+}
+h3{
+    color:rgb(238, 36, 238);
+    font-size: 20px;
+    display:flex;
+    justify-content: flex-start;
+}
+
+h2{
+    color: rgb(36, 191, 238);
+    font-size:3em;
+    display:flex;
+    justify-content: space-evenly;
+}
+
+.contact{
+    display:flex; 
+    justify-content: space-evenly;
+    font-weight: bold;
+    font-size: 30px;
```

Comment:
this styling is very similar to the styling in the `info` class, so consider giving them the same class names, maybe like `secondary-page-content` on top of the respective class names the divs already have, and give the new class the overlapping styling. then, for the differing class names of contact and info, you can put the colors there!